content/blog/productivity-porn.md

```diff
@@ -0,0 +1,36 @@
+
+Keep in mind that productivity is a personal thing. It can look like a variety of things: founding a startup, working out at the gym, building a side project, reading a book, growing a garden. Regardless, the key is that you benefit from doing the actual activity, but not from thinking about doing it. Hence the problem with productivity porn.
+
+I only realized I had this addiction when it was pointed out to me over and over again. [Start Marketing The Day You Start Coding](https://robwalling.com/assets/ebook.pdf) by Rob Walling, [The drawing advice that changed my life](https://www.youtube.com/watch?v=M6NsEDwHHiE&t=512s&ab_channel=struthless) by struthless, [How to get famous on twitter](https://podcasts.apple.com/us/podcast/how-to-get-famous-on-twitter-and-other-listener-questions/id1469759170?i=1000558020601) by the My First Million podcast, and [The Cooking Skill That Can't Be Taught](https://www.youtube.com/watch?v=gYwkKaK5yeQ&ab_channel=InternetShaquille) by Internet Shaquille all were telling me the message I needed to hear. They each said it in a different way, but the core of their message was the same. Stop thinking (reading, listening, watching etc.) about how to do something and just go do it. Or using my terminology, they were all saying that I was addicted to productivity porn[^1].
+
+Each time I'd hear their message I would be hit with an electric shock when I realized that they were right. I would immediately shut off my phone. For about a week after I would steer clear of any and all productivity porn, but inevitably I would find it weaseling its way back into my life.
+
+I find productivity porn to be a uniquely terrifying negative effect of social media for four reasons. While these reasons are also relevant to the other negative impacts of social media, I think they are particularly fitting for productivity porn.
+
+- **It saps my intrinsic motivation.** When I find myself wasting excessive amounts of time on social media my intrinsic motivation normally acts as a back stop. Eventually I will become tired of scrolling on social media and will go do something productive. However, productivity porn short circuits this by making me think I was being productive the entire time.
+- **It is very deceptive.** As far as I can tell I've been negatively impacted by this since at least highschool and I'm only realizing it right now. Looking back on it, how much time have I actually spent doing things versus just thinking about doing them.

```

Comment:
The last sentence seems like a question, but doesn't end in a question mark. If you want to keep it as a rhetorical question, I'd suggest finishing in a question mark. Otherwise, I'd suggest wording it slightly differently:
```suggestion
- **It is very deceptive.** As far as I can tell I've been negatively impacted by this since at least highschool and I'm only realizing it right now. Looking back, it's alarming to think about how much time I have spent actually doing things versus just thinking about doing them.
```